DuckDuckGo/TabSwitcherViewController.swift

```diff
@@ -250,13 +252,15 @@ extension TabSwitcherViewController: TabViewCellDelegate {
             collectionView.reloadData()
         } else {
             collectionView.performBatchUpdates({
+                isProcessingUpdates = true
                 delegate.tabSwitcher(self, didRemoveTab: tab)
                 currentSelection = tabsModel.currentIndex
                 collectionView.deleteItems(at: [IndexPath(row: index, section: 0)])
             }, completion: { _ in
                 guard let current = self.currentSelection else { return }
```

Comment:
Do you want to reset the flag in this return as well?

Reply:
Ah yes good catch 👍